specification/validated-relationships-service-api.yaml

```diff
@@ -1213,136 +1258,291 @@ components:
                 - outcome
               example: match
 
-    Patient:
+    ConsentBundleEntry: 
       type: object
-      description: The patient's details.
+      description: A Consent bundle entry.
```

Comment:
"A consent resource"?

Reply:
Or "Consent resource returned as a result of search"?

Reply:
Nah, just stick to the first suggestion. "A consent resource" feels more appropriate as the fact that it's a bundle entry doesn't _really_ matter to a consumer.

Reply:
... and the consumer knows they've searched for one.

Reply:
It's not a consent resource, it's a consent bundle entry (with `fullUrl` and `search` properties) and the Consent resource is  a child of this